book/doctrine.rst

```diff
@@ -386,9 +391,10 @@ Generating Getters and Setters
 
 Even though Doctrine now knows how to persist a ``Product`` object to the
 database, the class itself isn't really useful yet. Since ``Product`` is just
-a regular PHP class, you need to create getter and setter methods (e.g. ``getName()``,
-``setName()``) in order to access its properties (since the properties are
-``protected``). Fortunately, Doctrine can do this for you by running:
+a regular PHP class with ``protected`` properties, you need to create ``public``
```

Comment:
Why do we use `protected` properties at all? When we are overhauling this article, I suggest to switch to `private` properties too. What do you think?